aep/general/0134/aep.md.j2

```diff
@@ -57,7 +59,28 @@ separator, `snake_case` is used.
 
 {% tab oas %}
 
-**Note:** OAS example not yet written.
+```yaml
+paths:
+  /v1/publishers/{publisher}/books:
+    patch:
+      summary: Update a book
+      operationId: publishers.book:create
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Book'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Book'
+```
+
+- The method **must** adhere to the behavior specified in [IETF 7396 - JSON Merge Patch](https://datatracker.ietf.org/doc/html/rfc7396).
```

Comment:
That behavior corresponds with request media type `application/merge-patch+json`, which is not used above. Assuming the generalization to cover `application/json` as well is intentional, I have concerns about forward compatibility with a format that better handles merging arrays (i.e., by actually doing so rather than by replacement) or comprehends `null` differently. And at any rate, the divergence from RFC 7396 merits a specific mention.

Reply:
ah good point - Do you think we'd want to support both `merge-patch+json` and `json`? I'd imagine so from a user perspective - I'm not sure many will recall that detail about merge-patch.

re forward compatibility - are you thinking that if we want to support a new format, we could do so by adding another MIME type? Is there such a format we could start to look at now?

Reply:
> ah good point - Do you think we'd want to support both `merge-patch+json` and `json`? I'd imagine so from a user perspective - I'm not sure many will recall that detail about merge-patch.

I see no reason *not* to support `application/merge-patch+json` for requesting its behavior; it's extending that behavior to `application/json` that concerns me.

> re forward compatibility - are you thinking that if we want to support a new format, we could do so by adding another MIME type?

Yes, but if `application/json` today means JSON Merge Patch, then everybody is stuck with it forever, despite its obvious flaws.

> Is there such a format we could start to look at now?

Well, [JSON Patch](https://www.rfc-editor.org/rfc/rfc6902) already exists, but is not a great fit for AEP. And I have a draft of a "JSON List Patch" format that fixes JSON Patch's issues with future compatibility in mind, but it's not even public and I don't know if this group wants to pioneer in that direction.

The fundamental issue is that there is more than one JSON-expressible approach to patching a resource and the current state of the art isn't great, but picking a winner for silently specializing `application/json` is a **permanent** decision.

Reply:
Sorry it took me a while to round back.

> The fundamental issue is that there is more than one JSON-expressible approach to patching a resource and the current state of the art isn't great, but picking a winner for silently specializing application/json is a permanent decision.

Fair enough, I do see the point that setting a de-factor winner might not be productive.

It seems like there's universal agreement among those here that the guidance specifying support of json-merge-patch is valuable, so I'm going to shrink this PR to focus on that. We can discuss further formats of support in future PRs. How does that sound?

Reply:
I've updated the guidance and removed mention of `application/json` for now.